facebook/Makefile

```diff
@@ -133,6 +150,34 @@ $(WEIGHTS): $(TODAY)
 	  echo "WARNING: $${MSG}" >> $(MESSAGES); \
 	fi
 
+$(WEEKLY_WEIGHTS): $(TODAY)
+	# Want to schedule this to run every day. A pipeline run is triggered when new weekly weights files are available.
```

Comment:
Since this is going to run daily, instead of scheduling a separate cron job, can the `WEEKLY_WEIGHTS` target be added as a final [dependency of the normal pipeline instead](https://github.com/cmu-delphi/covidcast-indicators/blob/ndefries/add-weekly-weights-microdata/facebook/Makefile#L194)? @krivard

Reply:
And in that case would we still want separate messages and log files?

Reply:
Yeah, it makes sense to add it to the normal daily run, and in that case I agree we don't need new messages or log files.